Code/src/megadesk.cpp

```diff
@@ -69,26 +74,29 @@
 
 // constants related to presses/eeprom slots
 // (on attiny841: 512byte eeprom means max 255 slots)
-#define MIN_HEIGHT_SLOT 20
-#define MAX_HEIGHT_SLOT 22
-#define RIGHT_SLOT_START 32 // 0x20 in hex
-
 // EEPROM magic signature to detect if eeprom is valid
-#define EEPROM_LOCATION_SIG 0
+#define EEPROM_SIG_SLOT  0
 #define MAGIC_SIG 0x120d // bytes: 13, 18 in little endian order
+#ifdef MINMAX
+#define MIN_HEIGHT_SLOT  11
+#define MAX_HEIGHT_SLOT  12
+#endif
+#define RECALIBRATE      14 // nothing is stored there
+#define RESERVED_VARIANT 16 // reserved - deliberately empty
+#define BOTHBUTTON_SLOT  18 // store whether bothbuttons is enabled
+#define DOWN_SLOT_START  32 // 0x20 in hex offset for down button slots
```

Comment:
I'm not convinced these are the ideal slots/presses for:
min, max, recalibrate, bothbutton toggle
but they're easily shuffled around now...